src/routes/release.js

```diff
@@ -197,15 +197,15 @@ router.get(
       return res.redirect('/');
     }
 
-    let releaseNotes = release.body;
-    const parsed = semver.parse(version);
-    if (parsed.prerelease.length) {
-      releaseNotes = releaseNotes.split(new RegExp(`@${escapeRegExp(version.substr(1))}\`?.`))[1];
-    }
-    releaseNotes =
-      '# Release Notes\n' +
-      releaseNotes.replace(/# Release Notes for [^\r\n]+(?:(?:\n)|(?:\r\n))/i, '');
+    let releaseNotes = release && release.body ? release.body : 'No release notes available.';
+
+const parsed = semver.parse(version);
+if (parsed.prerelease.length) {
+  releaseNotes = releaseNotes.split(new RegExp(`@${escapeRegExp(version.substr(1))}\`?.`))[1] || releaseNotes;
+}
```

Comment:
If we already now know `releaseNotes` will be a valid string, we don't need `|| releaseNotes;`